### PR TITLE
Remove never-supported nodeFromPoint

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: #6854, #6768

### CODE REVIEW

File: api/DocumentOrShadowRoot.json [hunk 1]
```diff
@@ -467,106 +467,6 @@
...
-            },
-            "chrome_android": {
-              "version_added": false
-            },
-            "edge": {
-              "version_added": false
-            },
-            "firefox": {
-              "version_added": false,
-              "notes": "Implemented in Firefox but currently only works in chrome code."
```
Comment (foolip) [inline api/DocumentOrShadowRoot.json:485]: https://github.com/mozilla/gecko-dev/blob/f98dad153b59a985efd4505912588d4651033395/dom/webidl/DocumentOrShadowRoot.webidl#L18-L23 confirms this. There's a reference to https://github.com/w3c/csswg-drafts/issues/556 which is still open, there isn't anything to implement yet.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/nodeFromPoint and https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/nodesFromPoint would have to be archived. @sideshowbarker do you have the power to do this? (Not merging this yet.)

**Comment by sideshowbarker** [comment]
> https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/nodeFromPoint and https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/nodesFromPoint would have to be archived. @sideshowbarker do you have the power to do this?

No, I don’t — and not sure who else among BCD contributors might, except for @chrisdavidmills

**Comment by chrisdavidmills** [comment]
Yup, I can do this. Do you think archiving is the best idea? I could just delete them, or perhaps redirect them to the main interface page, to avoid the chance of broken links.

**Comment by foolip** [comment]
I don't know what's usually done with these pages, but I think @Elchi3 or @ddbeck helped me archive one in a previous instance.

We will need to archive or remove a lot of pages to land the [many PRs @saschanaz has sent](https://github.com/mdn/browser-compat-data/pulls/saschanaz) so it'd be great to pick an approach and coordinate a bit. Maybe like this:
- Someone reviews to confirm historical non-implementation of the removed entries
- That someone approves and ping a second reviewer who can archive/delete MDN pages
- That second reviewer does the MDN thing and then merges the PR

**Comment by ddbeck** [comment]
> it'd be great to pick an approach and coordinate a bit. Maybe like this:
> 
> * Someone reviews to confirm historical non-implementation of the removed entries
> * That someone approves and ping a second reviewer who can archive/delete MDN pages
> * That second reviewer does the MDN thing and then merges the PR

I agree with this and I like this as proposed. To fill in one detail: I like the idea of redirecting to the parent interface (if it exists). If there's no parent, then we'll need to take it on a case-by-case basis, perhaps with a bias toward deletion unless someone can come up with an affirmative case for preserving the page.

For pages that need redirects, I'm happy to be the default second reviewer for the MDN redirect + merge. For deletes/archives, I think that needs extra permissions. I guess we pull in @chrisdavidmills for those?

Any suggestions or objections before we just go for it?

**Comment by foolip** [comment]
Sounds great to me, if @chrisdavidmills is up for some page deletion duty :)

**Comment by chrisdavidmills** [comment]
> Sounds great to me, if @chrisdavidmills is up for some page deletion duty :)

Yup, definitely! 

I'd like to go with @ddbeck 's suggestion of redirecting to parent page if there is a sensible parent to redirect to, or deleting if not.

Aside from that, send me the list of pages and I can get to work.

**Comment by foolip** [comment]
I've summarized our process in https://github.com/mdn/browser-compat-data/pull/6854#issuecomment-718672722 and will start with this very PR.

**Review by foolip — APPROVED** [review]
These are Gecko APIs never exposed to web content. These pages need redirecting to the parent page:
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/nodeFromPoint
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/nodesFromPoint

Over to @ddbeck

**Review by ddbeck — APPROVED** [review]
Redirects done and merging. Thanks everyone for coming together for a good way to handle these. 👍

**Pull request merged**